src/components/DlCounters.vue

```diff
@@ -11,7 +11,11 @@
             >
                 <div :class="computeClass('item-content')">
                     <p :class="computeClass('item-value')">
-                        {{ item.value }}
+                        {{
+                            abbreviateNumbers
```

Comment:
Please change this to a computed getter as it will be more readable

Reply:
> Please change this to a computed getter as it will be more readable

This uses the value inside the v-for loop, so we can't use computed. Maybe a mthod?